src/components/dashjs-player/dashjs-player.tsx

```diff
@@ -66,12 +93,12 @@ export class DashjsPlayer {
           this.player.reset();
         }
         this.player = dashjs.MediaPlayer().create();
-        this.player.updateSettings(this.settings);
-        this.player.initialize(this.element.querySelector('#myMainVideoPlayer video'), LocalVariableStore.mediaUrl, event.detail.autoPlay == 'true');
+        this.player.initialize(this.element.querySelector('#myMainVideoPlayer video'), getMediaURL(), event.detail.autoPlay == 'true');
```

Comment:
you removed the update Settings call, this is wrong

Reply:
reverted, done

Reply:
getMediaUrl shouldn't be available anymore